crates/astria-sequencer/src/accounts/state_ext.rs

```diff
@@ -157,6 +158,48 @@ pub(crate) trait StateWriteExt: StateWrite {
         self.put_raw(nonce_storage_key(address), bytes);
         Ok(())
     }
+
+    #[instrument(skip(self))]
+    async fn increase_balance(
+        &mut self,
+        address: Address,
+        asset: asset::Id,
+        amount: u128,
+    ) -> Result<()> {
+        let balance = self
+            .get_account_balance(address, asset)
+            .await
+            .context("failed to get account balance")?;
+        self.put_account_balance(
+            address,
+            asset,
+            balance
+                .checked_add(amount)
+                .ok_or(anyhow!("account balance overflow"))?,
```

Comment:
Same as for the decrease_balance bit below, anyhow can be used directly on options:
```suggetion
                .context("adding to account balance failed due to overflow")?,
```